crates/config/src/sections/http.rs

```diff
@@ -124,6 +124,9 @@ pub enum BindConfig {
         /// Path to the socket
         #[schemars(with = "String")]
         socket: Utf8PathBuf,
+
+        /// Socket file mode
```

Comment:
Please add in the comment what form you expect this to be. You could also use [schemars' `example` attribute](https://graham.cool/schemars/v0/deriving/attributes/#example), which will appear in the generate JSONSchema